src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -292,12 +295,13 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
       .subscribe((planState) => {
         planID = planState.currentPlanId!;
```

Comment:
OOC, you've got planID and plan_id which makes reading a bit confusing.  (Yes, one is a number and one is a string - do you need both)?  Is there a chance that planID != int(plan_id) ?

Reply:
Oh thanks for highlighting this, the number plan_id was just to fill the requirements for a ProjectConfig, but I just realized it's not actually needed there (planning area id is saved in scenarioConfig, and ProjectConfig is going to be replaced in the next pr). Removed